main/cmds.go

```diff
@@ -124,8 +124,22 @@ func enablePre(ctx *log.Context, hEnv HandlerEnvironment, seqNum int) error {
 		return errors.Wrap(err, "failed to process sequence number")
 	} else if shouldExit {
```

Comment:
I suppose you should add tests to make sure that the new behavior you're adding is working as expected

Reply:
Tests are in this [repo](https://github.com/Azure/azure-extension-platform/blob/main/pkg/utils/utils_linux_test.go) 

Should I add similar ones here?